doc/proposal/expected/d0323r3.md

```diff
@@ -969,247 +1377,282 @@ template <class T, class E>
 
-*Throws*: Any exception thrown by the selected constructor of `T` or `unexpected<E>`
+*Throws*: Any exception thrown by the selected constructor of `T` or
+`unexpected<E>`
 
-*Remarks*: The expression inside noexcept is equivalent to: `is_nothrow_move_constructible<T>::value and 
-is_nothrow_move_constructible<E>::value`.
-This constructor shall not participate in overload resolution unless
-`is_move_constructible<T>::value and
+*Remarks*: The expression inside noexcept is equivalent to:
+`is_nothrow_move_constructible<T>::value and
+is_nothrow_move_constructible<E>::value`.  This constructor shall not
+participate in overload resolution unless `is_move_constructible<T>::value and
 is_move_constructible<E>::value`.
 
 ###########################################################################
 ```c++
-	template <class U, class G>
-	EXPLICIT expected(const expected<U,G>& rhs) noexcept(’see below’);
+    template <class U, class G>
+    explicit expected(const expected<U,G>& rhs) noexcept('see below');
 ```
 
-*Effects*: If `bool(rhs)` initializes the contained value as if direct-non-list-initializing an object of type `T` with the expression `*rhs`.
+*Effects*: If `bool(rhs)` initializes the contained value as if
+direct-non-list-initializing an object of type `T` with the expression `*rhs`.
 
-If `!bool(rhs)` initializes the contained value as if direct-non-list-initializing an object of type `unexpected<E>` with the expression `unexpected(rhs.error())`.
+If `!bool(rhs)` initializes the contained value as if
+direct-non-list-initializing an object of type `unexpected<E>` with the
+expression `unexpected(rhs.error())`.
 
 *Postconditions*: `bool(rhs) == bool(*this)`.
 
-*Throws*: Any exception thrown by the selected constructor of `T` or `unexpected<E>`
+*Throws*: Any exception thrown by the selected constructor of `T` or
+`unexpected<E>`
 
-*Remarks*: The expression inside noexcept is equivalent to: `is_nothrow_constructible<T, U&>::value and 
-is_nothrow_constructible<E, G&>::value`. 
-This constructor shall not participate in overload resolution unless `is_constructible<T, U&>::value` and
-`is_constructible<E, G&>::value`.
-The constructor is explicit if and only if 
-`is_convertible_v<U const&, T>` is false or `is_convertible_v<G const&, E> is false.
+*Remarks*: The expression inside noexcept is equivalent to:
+`is_nothrow_constructible<T, U&>::value and is_nothrow_constructible<E,
+G&>::value`.  This constructor shall not participate in overload resolution
+unless `is_constructible<T, U&>::value` and `is_constructible<E, G&>::value`.
+The constructor is explicit if and only if `is_convertible_v<U const&, T>` is
+false or `is_convertible_v<G const&, E> is false.
 
 ###########################################################################
 
 ```c++
-	template <class U, class G>
-	EXPLICIT expected(expected<U,G>&& rhs) noexcept(see below);
+    template <class U, class G>
+    explicit expected(expected<U,G>&& rhs) noexcept(see below);
```

Comment:
THe intent was EXPLICIT in upper case as we have for pair. The meaning of EXPLICIT differs. See #1600.